remote-deploy.sh

```
@@ -0,0 +1,141 @@
+# 기본 설정값
+SERVER_IP=""
+SERVER_USER="ubuntu"
+DEPLOY_PATH="/opt/summy_server"
+SSH_KEY="summy.pem"
+JAR_FILE="./api/build/libs/api-0.0.1-SNAPSHOT.jar"
+
+# 서버 설정 파일이 있으면 해당 파일에서 설정을 로드
+CONFIG_FILE="server-config.env"
+if [ -f "$CONFIG_FILE" ]; then
+  echo "서버 설정 파일 로드 중: $CONFIG_FILE"
+  source $CONFIG_FILE
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)